entity-framework/core/platforms/index.md

```diff
@@ -7,26 +7,54 @@ uid: core/platforms/index
 
 ## Overview
 
 The following table provides guidance for each .NET implementation:
 
-| .NET implementation                                                                                                  | Status                                                             | EF Core 1.x requirements                                                                                | EF Core 2.x requirements <sup>(1)</sup>                                                                 |
-|:---------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------|
-| **.NET Core** ([ASP.NET Core](../get-started/aspnetcore/index.md), [Console](../get-started/netcore/index.md), etc.) | Fully supported and recommended                                    | [.NET Core SDK 1.x](https://www.microsoft.com/net/core/)                                                | [.NET Core SDK 2.x](https://www.microsoft.com/net/core/)                                                |
-| **.NET Framework** (WinForms, WPF, ASP.NET, [Console](../get-started/full-dotnet/index.md), etc.)                    | Fully supported and recommended. EF6 also available <sup>(2)</sup> | .NET Framework 4.5.1                                                                                    | .NET Framework 4.6.1                                                                                    |
-| **Mono & Xamarin**                                                                                                   | In progress <sup>(3)</sup>                                         | Mono 4.6 <br/> Xamarin.iOS 10 <br/> Xamarin.Mac 3 <br/> Xamarin.Android 7                               | Mono 5.4 <br/> Xamarin.iOS 10.14 <br/> Xamarin.Mac 3.8 <br/> Xamarin.Android 7.5                        |
-| [**Universal Windows Platform**](../get-started/uwp/index.md)                                                        | EF Core 2.0.1 recommended <sup>(4)</sup>                           | [.NET Core UWP 5.x package](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform/) | [.NET Core UWP 6.x package](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform/) |
+| EF Core                       | 1.x    | 2.x        | 3.x             |
+|:------------------------------|:-------|:-----------|:----------------|
+| .NET Standard                 | 1.3    | 2.0        | 2.1             |
+| .NET Core                     | 1.0    | 2.0        | 3.0             |
+| .NET Framework<sup>(1)</sup>  | 4.5.1  | 4.7.2      | (not supported) |
```

Comment:
Isn't 2.x support 4.6.1 rather than 4.7.2, do we specify the latest or lowest version?

Reply:
Official guidance is to NOT use 4.6.1. See https://github.com/dotnet/standard/blob/master/docs/versions.md#net-standard-versions